internal/plugins/helm/v1/scaffolds/api.go

```diff
@@ -0,0 +1,134 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+Modifications copyright 2020 The Operator-SDK Authors
```

Comment:
This looks deliberate, but I am surprised that this file wouldnt be a full Operator-SDK copyright file

Reply:
The file is copied from KB and modified we need to respect the license. See: https://softwareengineering.stackexchange.com/questions/157968/how-to-manage-a-copyright-notice-in-an-open-source-project

Reply:
In that case this should be:
```
Original work Copyright 2019 The Kubernetes Authors.
Modified work Copyright 2020 The Operator-SDK Authors
```

Reply:
Along with any other file that was copied with modifications.